okhttp/src/jvmTest/java/okhttp3/internal/concurrent/TaskRunnerTest.kt

```diff
@@ -396,16 +396,15 @@ class TaskRunnerTest {
     assertThat(log).isEmpty()
 
     taskFaker.advanceUntil(100.µs)
-    assertThat(log).containsExactly(
+    assertThat(log).containsExactlyInAnyOrder(
         "one:run@100000 parallel=true",
         "two:run@100000 parallel=true",
         "three:run@100000 parallel=true"
     )
 
-    taskFaker.runTasks()
```

Comment:
I was able to get rid of this by introducing `TaskFaker.isRunningAllTasks`. This call was only introduced today and I wasn’t happy with it!